src/components/common/NoSSR.tsx

```diff
@@ -2,9 +2,13 @@ import dynamic from "next/dynamic";
 import React from "react";
 
 const NoSsr = ({ children }: { children: JSX.Element }) => (
+  // TOOD: Document: Why is a Fragment used.
```

Comment:
I made this comment because I was unaware of the reason why Fragment is being used; I wanted it to be explored and documented. Or changed to something standard.

I'm fairly sure `<></>` and `<React.Fragment></React.Fragment>` are the same thing; I guess it confused me that the `children` prop was a `JSX.Element`, which means it's a single element. Purposefully rendering a `Fragment` seems unusual, in my opinion.

Reply:
`<></>` and `<React.Fragment></React.Fragment>` are the same thing. But yea, you're right in that it's confusing that we'd be wrapping this in a fragment, as we'd just be wrapping a single element.